airbyte-integrations/connectors/source-shopify-singer/unit_tests/unit_test.py

```diff
@@ -22,5 +22,8 @@
 SOFTWARE.
 """
 
+import os
+
+
 def test_example_method():
-    assert True
+    assert os.path.commonpath(["/usr/lib", "/usr/local/lib"]) == "/usr"
```

Comment:
what is this change?

Reply:
There was no real tests and i just add something that use an import... not sure why i keep getting formatting back and forth with this empty file...